{"name": "Проверяем появление ошибки при авторизации с некорректным password", "status": "passed", "description": "Тестирование класса авторизации пользователя", "start": 1752511667251, "stop": 1752511667543, "uuid": "1dcaee9b-f69d-4d46-869e-abf06584d756", "historyId": "538a6ff7332a03548f1c11ede7c85b86", "testCaseId": "538a6ff7332a03548f1c11ede7c85b86", "fullName": "tests.test_login_user.TestLoginUser#test_login_error_to_auth_incorrect_email", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-PM8THT8"}, {"name": "thread", "value": "13836-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}